### PR TITLE
Feature/project description improvements

### PR DESCRIPTION
**Pull request by rubenb994**
## Description

Implemented Quill as WYSIWYG Editor for project add & edit.
Implemented localStorageUtils.
Fixed issue where beta banner was not dismissible

## Type of change

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce
Implemented Quill as WYSIWYG Editor for project add & edit.
1. Navigate to project add or edit
2. System display WYSIWYG editor

Implemented localStorageUtils.
1. None 

Fixed issue where beta banner was not dismissible
1. Click dismiss button on beta banner
2. Refresh
3. System does not display beta banner

Closes: #216 
Closes: #238 
Closes: #239

### CODE REVIEW

**Comment by rubenb994** [comment]
Found one issue with privacy consent, will fix after dinner

File: src/app/modules/project/details/details.component.html [hunk 1]
```diff
@@ -85,14 +85,16 @@ <h1 class="project-title">{{ project.name }}</h1>
           see more tags&hellip;
         </a>
       </div>
+      <!-- <p class="project-description large"> -->
```
Comment (wotwot563) [inline src/app/modules/project/details/details.component.html:88]: please remove commented out html

**Comment by Brend-Smits** [comment]
@rubenb994 when pressing enter in the editor I see these errors in my console: 
https://gyazo.com/59b663ee86e4c0a996ce4b6a7f8e23ba

**Comment by rubenb994** [comment]
> [h](https://gyazo.com/59b663ee86e4c0a996ce4b6a7f8e23ba)

Will look into it

File: src/app/modules/project/details/details.component.html [hunk 2]
```diff
@@ -85,14 +85,15 @@ <h1 class="project-title">{{ project.name }}</h1>
           see more tags&hellip;
         </a>
       </div>
+
       <p class="project-description large">
-        {{ project.description }}
+        <quill-view class="dex-quill-viewer" [content]="project.description" format="text"></quill-view>
```
Comment (StijnGroenen) [inline src/app/modules/project/details/details.component.html:90]: Just a question: Did you check if this makes XSS possible?
Quill has an option to sanitise HTML, but this is disabled by default. You are using the "text" format so I don't know if this is an issue.

Comment (rubenb994) [reply src/app/modules/project/details/details.component.html:90]: The following sanitizations take place:
- Quill itself performs sanitization on input
- Description content is also put through a sanitizer in the backend
- Angular escapes all content retrieved through the HttpClient
- Angular performs last sanitization before setting the content in the innerHTML

**Comment by rubenb994** [comment]
> Uploading very large images completely breaks the application, example is: https://prd-wret.s3.us-west-2.amazonaws.com/assets/palladium/production/s3fs-public/thumbnails/image/test-5mb.png
> 
> I think we should consider removing the image upload feature as it's very easy to break the service..

Will do, I'm dropping the video functionality as well

**Pull request merged**